.github/pull_request_template.md

```diff
@@ -31,5 +34,5 @@ Note that properly "linked issues" (either automatic links, or manual ones using
 * Which compiler / HPC you used to run the regression tests in the PR? 
 * Please provide the summary output of matrix.comp (_matrix.Diff.txt_, _matrixCompFull.txt_ and _matrixCompSummary.txt_):    
 Please indicate the expected changes in the outputs ([excluding the known list of non-identical tests](https://github.com/NOAA-EMC/WW3/wiki/How-to-use-matrix.comp-to-compare-regtests-with-master#4-look-at-results)).
-
+* Make sure you have added labels to the PR, including but not limited to _bug_, _documentation_, _duplicate_, _enhancement_, _invalid_, _mod_def change_, _out_grd change_, _out_pnt change_, _restart file change_, _new feature_, _Regression test_, _UFS_, ...
```

Comment:
So perhaps this should be, please add appropriate labels to indicate a change: 
_mod_def change_, _out_grd change_, _out_pnt change_, _restart file change_


And then maybe somewhere else we ask for labels such as _bug_ or _documentation_,  _new feature_   ?   Some of these labels were created for issues and some fit but some seem to be duplicates of each other  and maybe not all are necessary.  Hopefully someone isn't creating a new invalid PR for example.